src/util.cpp

```diff
@@ -416,6 +416,7 @@ static std::string ToDasher(const std::string &input) {
 
 std::string ConvertCase(const std::string &input, Case output_case,
                         Case input_case) {
+  if (output_case == Case::kKeep) return input;
   // The output cases expect snake_case inputs, so if we don't have that input
   // format, try to convert to snake_case.
   switch (input_case) {
```

Comment:
We might need to include input_case kKeep case. Though a warning or something?

Reply:
I don't understand, can you rephrase?

Reply:
if someone set output_case = snake and input_case = keep, it would fail parse correctly. I was saying we should handle if input_case is keep and return an error or something.

Reply:
Gotcha, added a warning